core/src/test/scala/integration/kafka/api/IntegrationTestHarness.scala

```diff
@@ -62,6 +62,7 @@ abstract class IntegrationTestHarness extends KafkaServerTestHarness {
   }
 
   override def generateConfigs: Seq[KafkaConfig] = {
+
```

Comment:
extra newline

Reply:
This wasn't done by me, it was something that David removed in his PR and I reverted it. But sure we can remove it.